### ISSUE TITLE
[Question] Why are losses negative?

### ISSUE THREAD

**Issue by neuronflow** [JunMa11/SegLoss#8]
Many losses in the end `return -loss` , e.g. SoftDiceLoss or Tversky:
https://github.com/JunMa11/SegLoss/blob/131663994df22efb907e7ae897a39efe628518fc/losses_pytorch/dice_loss.py#L291

Why are the losses returned negative?

**Comment by JunMa11** [comment]
Hi @neuronflow ,

This is because `dice_loss = 1 - dice`.
`1` can be omitted because constant does not affect backpropagation.
So, we use `-dice` as the loss.

**Comment by neuronflow** [comment]
Thanks for the quick response and why is this only done for a few of the losses? :)
E.g. Generalized Dice should follow the same results?

**Comment by JunMa11** [comment]
Yes. I standardize the GDice loss too.
 `1 - GDice` -> `- GDice`.
